package.json

```diff
@@ -7,7 +7,8 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack --mode=development",
     "server": "webpack-dev-server --inline --progress --mode=development",
-    "start": "npm run server"
+    "start": "npm run server",
```

Comment:
Should this be 'yarn run server'?

Reply:
That was already there, think that's a question for @ltwheeler?